js/ui/html_editor/modules/tableResizing.js

```diff
@@ -0,0 +1,524 @@
+            const $nextLineElements = this._getLineElements(frame.$table, index + 1);
+
+            if(isCurrentColumnHasEnoughPlace && isNextColumnHasEnoughPlace) {
+                $lineElements.each((i, element) => {
+                    $(element).attr(directionInfo.positionStyleProperty, currentLineNewSize + 'px');
+                });
+
+                if(this._nextLineSize) {
+                    $nextLineElements.each((i, element) => {
+                        $(element).attr(directionInfo.positionStyleProperty, nextColumnNewSize + 'px');
+                    });
+                }
+            }
+        } else {
+            const newHeight = Math.max(currentLineNewSize, this._minRowHeight);
+            $determinantElements.eq(index).attr(directionInfo.positionStyleProperty, newHeight + 'px');
+        }
+
+        this._updateFramePosition(frame.$table, frame.$frame);
+    }
+
+    _isLastColumnResizing({ $determinantElements, index }) {
+        return !isDefined($determinantElements[index + 1]);
+    }
+
+    _getBoundaryConfig(options) {
+        const result = {};
+
+        if(options.direction === 'vertical') {
+            result.boundary = options.frame.$table;
+            result.boundOffset = {
+                bottom: hasWindow() ? -$(getWindow()).height() : -$(this._quillContainer).outerHeight,
+                top: 0,
+                left: 0,
+                right: 0
+            };
+        } else {
+            if(!this._isLastColumnResizing(options)) {
+                result.boundary = options.frame.$table;
+            } else {
+                result.boundary = $(this._quillContainer);
+            }
+        }
+
+        return result;
+    }
+
+    _createDraggableElement(options) {
+        const boundaryConfig = this._getBoundaryConfig(options);
+
+        const config = {
+            contentTemplate: null,
+            allowMoveByClick: false,
+            dragDirection: options.direction,
+            onDragMove: ({ component, event }) => {
+                this._dragMoveHandler(event, options);
+            },
+            onDragStart: () => {
+                this._dragStartHandler(options);
+            },
+            onDragEnd: () => {
+                this._tableLastWidth(options.frame, options.frame.$table.outerWidth());
+                this._updateFramesPositions();
+                this._updateFramesSeparators();
+            }
+        };
+
+        extend(config, boundaryConfig);
+
+        this._currentDraggableElement = this.editorInstance._createComponent(options.lineSeparator, Draggable, config);
+    }
+
+    _fixColumnsWidth($table) {
+        const determinantElements = this._getTableDeterminantElements($table);
+
+        each(determinantElements, (index, element) => {
+            const columnWidth = $(element).outerWidth();
+            $(element).attr('width', columnWidth >= DEFAULT_MIN_COLUMN_WIDTH ? columnWidth : DEFAULT_MIN_COLUMN_WIDTH);
```

Comment:
`Math.max`?

Reply:
you set this attribute to string width `px` in another places, but to number here